allenact/setup.py

```diff
@@ -3,20 +3,61 @@
 
 from setuptools import find_packages, setup
 
+
+def parse_req_file(fname, start=None):
+    reqs = {} if start is None else start
+    cline = None
+    with open(fname, "r") as f:
+        for line in f.readlines():
+            line = line[:-1]
+            if len(line) == 0:
+                continue
+            if line[0] == "[":
+                cline = line[1:-1]
+                reqs[cline] = []
+            else:
+                if cline is not None:
+                    reqs[cline].append(line)
+    return reqs
+
+
+def get_version(fname):
+    res = "UNK"
+    with open(fname, "r") as f:
+        for line in f.readlines():
+            line = line[:-1]
+            if line.startswith("Version:"):
+                res = line.replace("Version:", "").strip()
+                break
+    return res
```

Comment:
We should throw an error if the version is unknown?

Reply:
👍